src/configuration/api/api.configuration.ts

```diff
@@ -2,7 +2,7 @@ export const API = {
   FILE_UPLOAD: "/api/v3/file/upload/school/SCHOOLID/boardnodes/CONTEXTID",
   FILE_DELETE: "/api/v3/file/delete/FILERECORD_ID",
   FILE_RESTORE: "/api/v3/file/restore/FILERECORD_ID",
-  LOGIN_REDIRECT: "/login?redirect=/tldraw?roomName=ROOMID",
+  LOGIN_REDIRECT: "/login?redirect=/tldraw?parentId=ROOMID",
```

Comment:
```suggestion
  LOGIN_REDIRECT: "/login?redirect=/tldraw?parentId=PARENTID",
```
Hier würd ichs dann auch noch umbenennen. Musste dann auch in redirectUtils anpassen.

Reply:
Fixed by applying suggested change!